allenact_plugins/manipulathor_plugin/manipulathor_environment.py

```diff
@@ -120,17 +186,17 @@ def start(
 
         self.controller = self.create_controller()
 
-        if (
-            self._start_player_screen_height,
-            self._start_player_screen_width,
-        ) != self.current_frame.shape[:2]:
-            self.controller.step(
-                {
-                    "action": "ChangeResolution",
-                    "x": self._start_player_screen_width,
-                    "y": self._start_player_screen_height,
-                }
-            )
+        # if (
+        #     self._start_player_screen_height,
+        #     self._start_player_screen_width,
+        # ) != self.current_frame.shape[:2]:
+        #     self.controller.step(
+        #         {
+        #             "action": "ChangeResolution",
+        #             "x": self._start_player_screen_width,
+        #             "y": self._start_player_screen_height,
+        #         }
+        #     )
```

Comment:
Can be deleted?

Reply:
Yes. I commented it because originally the controller's height/width is same as `DepthSensor` (224). So these code won't be run. But when I run visualization script with higher resolution, these code reset the resolution, and actually redundant as the `DepthSensor` will also do the rescaling for the agent.